Consolidation OOP/requirements.txt

```diff
@@ -0,0 +1,5 @@
+pyyaml
+numpy
+matplotlib 
+Pillow
+glob
```

Comment:
glob is part of the Python standard library, it does not need to be installed.
Right now, trying to install `requirements.txt` using pip yields and Error because of glob. Please remove it.